packages/bottlerocket-licenses/bottlerocket-licenses.spec

```diff
@@ -0,0 +1,32 @@
+%global _cross_first_party 1
+%global debug_package %{nil}
+
+Name: %{_cross_os}bottlerocket-licenses
+Version: 0.0
+Release: 0%{?dist}
+Summary: bottlerocket-licenses
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source100: COPYRIGHT
+Source101: LICENSE-MIT
+Source102: LICENSE-APACHE
```

Comment:
I would just put these files in the same directory as `os.spec` and include them in that package.